master/static/srv/proto_get_trials_plus.sql

```diff
@@ -0,0 +1,125 @@
+WITH const AS (
+  SELECT config->'searcher'->>'metric' AS metric_name,
+    (
+      SELECT CASE
+          WHEN coalesce(
+            (
+              config->'searcher'->>'smaller_is_better'
+            )::boolean,
+            true
+          ) THEN -1 -- so we can order by DESC to get the highest value
```

Comment:
Not sure this comment is necessary; this pattern is common enough already.

Reply:
this one is the reverse of other uses of this pattern that's why I added the comment kind of explaining why it's this way (DESC seems like a better fit). with that said I'm good with removing it : )

Reply:
Ah, I see. I would say to stick with the usual direction, then. No more reason to switch it here than anywhere else, as far as I can see.